vowpalwabbit/parse_args.cc

```diff
@@ -1321,6 +1321,8 @@ void parse_sources(vw& all, io_buf& model, bool skipModelLoad)
 {
   if (!skipModelLoad)
     load_input_model(all, model);
+  else
+    model.close_file();
```

Comment:
I'm surprised this didn't have a negative affect right now. Does this solve your exception?

Reply:
Hi @eisber . Apologies for the late reply. Yes, this solved the exception :) .